.circleci/config.yml

```diff
@@ -2,98 +2,68 @@ version: 2
 
 defaults: &defaults
   docker:
-    - image: codegram/decidim:test-latest
+    - image: decidim/decidim:dev-latest
       environment:
         SIMPLECOV: true
         DATABASE_USERNAME: postgres
         FAIL_FAST: true
     - image: postgres
       environment:
         POSTGRES_USER: postgres
-  working_directory: /decidim
+  working_directory: /app
```

Comment:
Why renaming this to `/app`? Isn't this folder supposed to hold decidim's code and not an actual application?

Reply:
It makes sense! I changed the docker repo's location to `/code` (which can hold any kind of code) and set this to `/decidim` to make things clearer 💃